public/data/dialog_data.csv

```diff
@@ -0,0 +1,5 @@
+header,title,class,text,source,alt_text
+Item,Item A,item_a,This is item A,image_a.png,This is an image of the letter a
+Item,Item B,item_b,This is item B,image_b.png,This is an image of the letter b
+Item,Item C,item_c,This is item C,image_c.png,This is an image of the letter c
+Item,Item D,item_d,This is item D,image_d.png,This is an image of the letter d
```

Comment:
Let's move this to s3 as the standard to avoid committing data files. Even though this one is super small, if the pattern is re-used doing this from the get-go will be beneficial

Reply:
Same comment for the images. We could skip the pipeline for submitting to s3 in this case, but omitting will keep the repo lighter.

Reply:
I can't log in to aws today, but will make this change when I can

Reply:
I just put it in there at https://labs.waterdata.usgs.gov/visualizations/data/dialog_data.csv

Reply:
and the images with the same file names at https://labs.waterdata.usgs.gov/visualizations/images/

Reply:
oh, awesome. thanks! I'll update the code to use them now